numpy/linalg/linalg.py

```diff
@@ -21,13 +21,14 @@
 from numpy.core import (
     array, asarray, zeros, empty, empty_like, intc, single, double,
     csingle, cdouble, inexact, complexfloating, newaxis, all, Inf, dot,
-    add, multiply, sqrt, fastCopyAndTranspose, sum, isfinite,
+    add, multiply, sqrt, sum, isfinite,
     finfo, errstate, geterrobj, moveaxis, amin, amax, product, abs,
     atleast_2d, intp, asanyarray, object_, matmul,
     swapaxes, divide, count_nonzero, isnan, sign, argsort, sort,
     reciprocal
 )
 from numpy.core.multiarray import normalize_axis_index
+from numpy.core.multiarray import _fastCopyAndTranspose as _fastCT
```

Comment:
To ease the eventual removal, shouldn't the internal usages of the C function here be converted to the recommended array.T.copy() replacement?

Reply:
I had initially planned on only deprecating the Python function, but since the plan is to deprecate the C function as well I agree this is the right call. I'm happy to do so in this PR since there seems to be consensus that deprecating the C funciton as well is desirable.